README.md

```diff
@@ -2,48 +2,38 @@
 
 How the .gov domain space is doing at best practices and federal requirements.
 
-A static website, with front-end assets managed by npm and Bower.
-
-## Install
+## Setup
 
-* Install [node and npm](https://nodejs.org/download/) directly.
-* Install [bower](http://bower.io/) through npm:
+* Have Bundler installed:
 
-```
-npm install -g bower
+```bash
+gem install bundler
 ```
 
-* Install the [Chrome LiveReload extension](https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en).
-
-## Setup
-
 * Run the following commands to install the necessary packages:
 
 ```bash
-npm install
-bower install
+bundle install
 ```
 
-* Now you should be able to run `gulp` and open the `index.html` file in your favorite browser.
+* Now you can run the app:
 
 ```bash
-gulp
+make run
 ```
```

Comment:
It might be good to add in something in here on the `make` commands provided and what they do. For example, if someone is working on the styles and needs to compile the sass. `make run` is not currently setup to compile and will not show the updated styles.